### PR TITLE
[PR] Improve style and layout of admin screen

### PR DESCRIPTION
**Pull request by jeremyfelt**

### CODE REVIEW

**Comment by jeremyfelt** [comment]
@philcable #reviewmerge?

File: includes/class-wsuwp-graduate-degree-programs.php [hunk 1]
```diff
@@ -290,12 +290,14 @@ public function display_factsheet_primary_meta_box( $post ) {
...
+				// If no fields have been added, provide an empty field by default.
+				if ( 0 === count( $field_data ) ) {
+					echo '<span class="factsheet-' . esc_attr( $meta['type'] ) . '-field">';
 
-				?><input type="text" name="<?php echo esc_attr( $key ); ?>[]" value="" /></span><?php
+					?><input type="text" name="<?php echo esc_attr( $key ); ?>[]" value="" /></span><?php
+				}
 
-				echo '<span class="factsheet-add-' . esc_attr( $meta['type'] ) . '">Add</span></div>';
+				echo '<input type="buttom" class="add-factsheet-' . esc_attr( $meta['type'] ) . '-field button" value="Add" /></div>';
```
Comment (philcable) [inline includes/class-wsuwp-graduate-degree-programs.php]: s/buttom/button for the input type.

**Review by philcable — CHANGES_REQUESTED** [review]
Looking good! I spotted one typo that should be fixed.

Unrelated to this PR, there's an extra `"` at the end of the div tag on line 261 in `includes/class-wsuwp-graduate-degree-programs.php`.

File: includes/class-wsuwp-graduate-degree-programs.php (hunk 1 above)
Comment (jeremyfelt) [reply includes/class-wsuwp-graduate-degree-programs.php]: Ha! I'm pretty awesome. Thanks! :)

**Comment by jeremyfelt** [comment]
@philcable Updated, #reviewmerge?

**Review by philcable — COMMENTED** [review]
👍

**Review by philcable — APPROVED** [review]

**Pull request merged**